src/System.Data.SqlClient/tests/ExceptionTest.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections;
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class ExceptionTest
+    {
+        // test connection string
+        string conn_string = "server=tcp:server,1432;database=test;uid=admin;pwd=SQLDB;connect timeout=60;";
+
+        // data value and server consts
+        private const string badServer = "NotAServer";
+        private const string sqlsvrBadConn = "A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)";
+        private const string execReaderFailedMessage = "ExecuteReader requires an open and available Connection. The connection's current state is closed.";
+        private const string orderIdQuery = "select orderid from orders where orderid < 10250";
+
+
+        [Fact]
```

Comment:
It is preferable to put this fact on the test functions instead of 1 function called 3 tests.

The reason behind it so that it will be easier to pin point the error directly from XUnit. Instead of dig through callstack to figure out which test are actual failing.
e.g
[Fact]
ExceptionTests();

[Fact]
VariousExceptionTests();